### PR TITLE
kicad-lite-nightly: Add latest version

### PR DESCRIPTION
**Pull request by tech189**
Relates to #258. Renamed to keep `-nightly` at the end of an app's name. Added a Last Modified check to checkver, which should catch when the latest version is updated but not increased in version number.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/kicad-lite-nightly.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://kicad-downloads.s3.cern.ch/windows/nightly/kicad-nightly-r27528.623dc8980f-x86_64-lite.exe#/dl.7z",
+            "hash": "03b6db02695f9f759878555adc1b298b4f543b050386afe249359ac676a7ba55"
+        }
+    },
+    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse",
+    "bin": [
+        [
+            "bin\\kicad.exe",
```
Comment (rashil2000) [inline bucket/kicad-lite-nightly.json:15]: CLI args?

Comment (tech189) [reply bucket/kicad-lite-nightly.json:15]: You can load a project file from the command line.

Comment (rashil2000) [reply bucket/kicad-lite-nightly.json:15]: In that case, kicad-nightly is fine, so that people can install stable and nightly side by side

Comment (tech189) [reply bucket/kicad-lite-nightly.json:15]: Sure thing, done.

**Pull request merged**